metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py

```diff
@@ -168,23 +168,18 @@ def external_url_defaults_to_api_config_base_url(
     ) -> Optional[str]:
         return v or values.get("base_url")
 
-    @validator("platform_instance")
-    def platform_instance_not_supported(cls, v: Optional[str]) -> Optional[str]:
-        if v is not None:
-            raise ConfigurationError("Looker Source doesn't support platform instances")
-        return v
-
 
 @platform_name("Looker")
 @support_status(SupportStatus.CERTIFIED)
 @config_class(LookerDashboardSourceConfig)
 @capability(SourceCapability.DESCRIPTIONS, "Enabled by default")
-@capability(SourceCapability.PLATFORM_INSTANCE, "Enabled by default")
+@capability(SourceCapability.PLATFORM_INSTANCE, "Not supported", supported=False)
 @capability(
     SourceCapability.OWNERSHIP, "Enabled by default, configured using `extract_owners`"
 )
 @capability(
-    SourceCapability.USAGE_STATS, "Can be enabled using `extract_usage_history`"
+    SourceCapability.USAGE_STATS,
+    "Enabled by default, configured using `extract_usage_history`",
```

Comment:
What does this decorator actually do? Does this update the docs or something?

Reply:
Yup it feeds into our doc generation system